plan/config_test.go

```diff
@@ -143,131 +143,115 @@ func TestParseSkipBehavior(t *testing.T) {
-						AxisMatch{
+						AxisMatcher{
 							Name:  "server",
 							Value: "b",
 						},
 					},
 				},
 			},
 			wantBehaviorName: "foo",
-			wantError:        nil,
 		},
 		{
+			desc: "multiple filters",
 			give: "x=a:b,c:d",
-			desc: "multiple filters.",
 			wantFilters: []Filter{
 				{
-					AxisMatches: []AxisMatch{
-						AxisMatch{
+					Matchers: []AxisMatcher{
+						AxisMatcher{
 							Name:  "a",
 							Value: "b",
 						},
 					},
 				},
 				{
-					AxisMatches: []AxisMatch{
-						AxisMatch{
+					Matchers: []AxisMatcher{
+						AxisMatcher{
 							Name:  "c",
 							Value: "d",
 						},
 					},
 				},
 			},
 			wantBehaviorName: "x",
-			wantError:        nil,
 		},
 		{
+			desc: "single filter testing name separation",
 			give: "foo=client:weird:name+server:b",
-			desc: "single filter testing name separation.",
 			wantFilters: []Filter{
 				{
-					AxisMatches: []AxisMatch{
-						AxisMatch{
+					Matchers: []AxisMatcher{
+						AxisMatcher{
 							Name:  "client",
 							Value: "weird:name",
 						},
-						AxisMatch{
+						AxisMatcher{
 							Name:  "server",
 							Value: "b",
 						},
 					},
 				},
 			},
 			wantBehaviorName: "foo",
-			wantError:        nil,
 		},
 		{
-			give:             "x=a:b,,c:d",
-			desc:             "invalid filters",
-			wantFilters:      nil,
-			wantBehaviorName: "",
-			wantError:        fmt.Errorf("match \"\" in input \"x=a:b,,c:d\" is not of form 'key:value'"),
+			desc:      "invalid: empty filter",
+			give:      "x=a:b,,c:d",
+			wantError: fmt.Errorf("invalid matcher \"\" in input \"x=a:b,,c:d\" is not of form 'key:value'"),
 		},
 		{
-			give:             "x",
-			desc:             "invalid filters",
-			wantFilters:      nil,
-			wantBehaviorName: "",
-			wantError:        fmt.Errorf("missing '=' in the input: \"x\""),
+			desc:      "invalid filters",
+			give:      "x",
+			wantError: fmt.Errorf("missing '=' in the input: \"x\""),
 		},
 		{
-			give:             "  x    ",
-			desc:             "invalid filters",
-			wantFilters:      nil,
-			wantBehaviorName: "",
-			wantError:        fmt.Errorf("missing '=' in the input: \"  x    \""),
+			desc:      "invalid filters",
+			give:      "",
+			wantError: fmt.Errorf("missing '=' in the input: \"\""),
 		},
 		{
-			give:             "x=:",
-			desc:             "invalid filters",
-			wantFilters:      nil,
-			wantBehaviorName: "",
-			wantError:        fmt.Errorf("match \":\" in input \"x=:\" is empty"),
+			desc:      "invalid filters",
+			give:      "  x    ",
+			wantError: fmt.Errorf("missing '=' in the input: \"  x    \""),
 		},
 		{
-			give:             "x=",
-			desc:             "invalid filters",
-			wantFilters:      nil,
-			wantBehaviorName: "",
-			wantError:        fmt.Errorf("match \"\" in input \"x=\" is not of form 'key:value'"),
+			desc:      "invalid: empty matcher",
+			give:      "x=:",
+			wantError: fmt.Errorf("invalid matcher \":\": axis name and value are required"),
```

Comment:
If you need to use quotes inside strings, use the ` form

    `invalid matcher ":": axis name and value are required`

This is the same as what you wrote except it's quite a lot more readable.